src/System.IO.FileSystem/tests/Directory/GetFiles.cs

```diff
@@ -164,5 +166,24 @@ public override string[] GetEntries(string path, string searchPattern, SearchOpt
         {
             return Directory.GetFiles(path, searchPattern, option);
         }
+
+        [Theory, MemberData(nameof(TrailingSeparators))]
+        public void DirectoryWithTrailingSeparators(string trailing)
```

Comment:
This test is naturally running and passing on NETFX right? Thus proving you are matching NETFX beahvior.

Reply:
That is the idea.